### DISCUSSION TITLE
Add from a compiled file (go)

### DISCUSSION THREAD

**Discussion by isaacismaelx14** [Homebrew/discussions#3730] [Casks] [answered]
Hey, everyone! 
I'm working with go-lang and I'm trying to add my compiled package homebrew. I just want a quick way to install my package.  I already know about `brew create --go` but I need to send the compiled package and not the source-code. How can I add it?

**Comment by SMillerDev** [comment, marked as answer]
Formula are supposed to contain source build information. If you want to distribute a binary use `brew create --cask`
